### PR TITLE
feat: (ecs7) change billboard approach

### PR DESCRIPTION
**Pull request by leanmendoza**
## What does this PR change?
Change the approach of Billboard in the ECS7. With this, a system is iterating every billboard and updates each frame the game object transform. The previous approach only waited for the camera changes to update the transform and it worked wrong if the entity updated its transform.

### CODE REVIEW

File: unity-renderer/Assets/DCLPlugins/ECS7/Systems/BillboardSystem/ECSBillboardSystem.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+namespace ECSSystems.BillboardSystem
+{
+    public class ECSBillboardSystem : IDisposable
+    {
+        
+        private class State
+        {
+            public ECSComponent<PBBillboard> billboards;
+            public DataStore_Camera camera;
+        }
```
Comment (AjimenezDCL) [inline unity-renderer/Assets/DCLPlugins/ECS7/Systems/BillboardSystem/ECSBillboardSystem.cs:16]: Why are we wrapping `billboards` and `camera` on a class? it seems like an unnecessary allocation. We could have some readonly fields instead.

**Review by AjimenezDCL — APPROVED** [review]
Approving with a small request.

File: unity-renderer/Assets/DCLPlugins/ECS7/Systems/BillboardSystem/ECSBillboardSystem.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                billboards = billboards,
+                camera = camera
+            };
+
+        }
+
+        public void Dispose()
+        {
+        }
+
```
Comment (pbosio) [inline unity-renderer/Assets/DCLPlugins/ECS7/Systems/BillboardSystem/ECSBillboardSystem.cs:33]: If no disposing is needed, we could remove this `IDisposable` implementation

**Comment by popuz** [comment]
hi @leanmendoza, since it will be included in the release, are there some steps to test it (for QA)?

**Comment by leanmendoza** [comment]
> hi @leanmendoza, since it will be included in the release, are there some steps to test it (for QA)?

Sure, although there isn't any `ECS7` scene deployed, you can test here: http://playground.decentraland.org/?&renderer-branch=
1. Open the playground
2. After loading, go to Examples>Billboard
3. Click on Run
you should see how the billboard example works fine.

**Pull request merged**